### DISCUSSION TITLE
dvas color palette

### DISCUSSION THREAD

**Discussion by fpavogt** [MeteoSwiss/dvas#139] [Ideas]
Hi @MCH-MDA/dvas-aficionados, 

I suggest that we keep discussing the dvas color palette here. This should be a good place to keep coming back at it over the coming months. 

After discussion and testing with @modolol, we realized that using a single color palette for *all* dvas plots may be of **de-service** to their scientific clarity. On the one hand, we need a set of 10 clearly distinguishable colors for multi-sondes flights. On the other, we should have a continuous colormap for continuous variables. 

The question of the B&W compatibility is also posing serious challenges/restrictions on the color choices. This would be useful if people print dvas plots in B&W. But realistically, most people will just look at them on a screen. In the end, dropping the B&W requirement may be result in a more pleasing set of plots.

You should all feel free to share ideas in the discussions below. We have a few months to settle on something that pleases us all :)

**Feel free to use the *like* buttons to express your love/hate for specific options !**

:bulb: Dropping **png** files in replies will display the images directly, without the need to download them first !

**Comment by fpavogt** [comment]
First suggestion, which is B&W compatible.

<img width="1868" alt="Screenshot 2021-05-05 at 09 40 53" src="https://user-images.githubusercontent.com/13974615/117110173-066d9580-ad86-11eb-8593-d9c74eea7f3a.png">

**Comment by fpavogt** [comment]
Second suggestion, this time dropping the B&W compatibility, and with a distinct set of 10 brights colors vs a smooth colormap.
**This will become the dvas default until we agree otherwise.**

<img width="1528" alt="Screenshot 2021-05-05 at 09 37 15" src="https://user-images.githubusercontent.com/13974615/117110285-2ef58f80-ad86-11eb-95a1-31f371892bf9.png">

**Reply by g-martucci** [reply]
Hi @fpavogt, first thanks for the work you do and the different options amongst which we can choose. I'm deeply in favor of a B&W independent option. It would be an anachronistic choice, a lot of work for a little profit. On the other side, having shared the entirety of my physics studies with my colorblind friend, I've learned how important is to have colorblind-compatible color palettes. And the second option, is in my opinion the real hell for colorblinds. I've sent the file to this friend for a test and I'll get back to you.

**Reply by fpavogt** [reply]
Sounds good. I've replaced the initial proposition with a tweaked version: same colors, but this time also with the "uncertainties" areas shown.

**Reply by g-martucci** [reply]
ok, that's much much better!! As a general rule, the luminance as perceived by the human eye grows at different speed for red, green and blue colors. Green is the one that varies the most and blue is the least. In the choice before, the component that changes the least is exactly the green, which makes almost impossible for a colorblind to distinguish the colors. 
![RGB_vs_luminance](https://user-images.githubusercontent.com/65007375/117114000-8f3b0000-ad8b-11eb-8bba-08bd15a946f2.jpg)
